docs/src/examples/components/Tooltip/Variations/TooltipExamplePosition.shorthand.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Button, Grid, Tooltip, Alignment, Position } from '@stardust-ui/react'
+import { useBooleanKnob, useSelectKnob } from '@stardust-ui/docs-components'
+
+const TooltipExamplePosition = () => {
+  const [open] = useBooleanKnob({ name: 'open-s', initialValue: true })
```

Comment:
Looks that `Tooltip` in this example should be always opened

```suggestion
-  const [open] = useBooleanKnob({ name: 'open-s', initialValue: true })
```

Reply:
Agreed, we have a separate example for the open prop.